project-auth/src/test/scala/io/renku/projectauth/ProjectAuthServiceSpec.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.renku.projectauth
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+import cats.effect.kernel.Resource
+import cats.effect.testing.scalatest.AsyncIOSpec
+import fs2.Stream
+import io.renku.generators.Generators.Implicits._
+import io.renku.graph.model.RenkuUrl
+import io.renku.graph.model.persons.GitLabId
+import io.renku.graph.model.projects.Visibility
+import io.renku.triplesstore.client.util.JenaContainerSpec
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should
+import org.typelevel.log4cats.Logger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+
+class ProjectAuthServiceSpec extends AsyncFlatSpec with AsyncIOSpec with JenaContainerSpec with should.Matchers {
+  implicit val logger:   Logger[IO] = Slf4jLogger.getLogger[IO]
+  implicit val renkuUrl: RenkuUrl   = RenkuUrl("http://localhost/renku")
+
+  def withProjectAuthService: Resource[IO, ProjectAuthService[IO]] =
+    withDataset("projectauth").map(ProjectAuthService[IO](_, renkuUrl))
+
+  it should "add data" in {
+    withProjectAuthService.use { s =>
+      for {
+        data <- Generators.projectAuthDataGen.asStream.toIO.take(20).compile.toVector
+        _ <- Stream
+               .emits(data)
+               .through(s.updateAll)
+               .compile
+               .drain
+
+        n <- s.getAll(15).compile.toVector
+        _ = n shouldBe data.sortBy(_.path)
```

Comment:
Is that the Project `path`? If so I think it would be good to rename it to `slug`

Reply:
Oops, yes! Must be changed